ironfish/src/rpc/routes/accounts/getNotes.ts

```diff
@@ -50,10 +51,15 @@ router.register<typeof GetAccountNotesRequestSchema, GetAccountNotesResponse>(
     const account = getAccount(node, request.data.account)
 
     const responseNotes = []
+    const stream = request.data?.stream
 
     for await (const { transaction } of account.getTransactions()) {
       const notes = await getTransactionNotes(account, transaction)
-      responseNotes.push(...notes)
+      if (stream) {
+        request.stream({ account: account.displayName, notes: [...notes] })
```

Comment:
Is `[...notes]` needed, or does `notes` work? 

```suggestion
        request.stream({ account: account.displayName, notes })
```

Reply:
`notes` returned from `getTransactionNotes` is readonly, while `notes` in `GetAccountNotesResponse` is mutable so I used `note: [...notes]`. Is there a better way to get around this?